act/retrievals/cbh.py

```diff
@@ -34,7 +39,13 @@ def generic_sobel_cbh(
     var_thresh : float
         Thresholding for variable if needed.
     fill_na : float
-        What to fill nans with in DataArray if any.
+        Value to fill nans with in DataArray if any.
+    uniform_filter : boolean
```

Comment:
Can you rename this to `filter_type` and set it to `uniform`? That way in the future, if we want to add additional filter types, we can?

Reply:
Done!